src/azul/openapi/schema.py

```diff
@@ -220,3 +309,27 @@ def make_type(t: TYPE) -> JSON:
         return t
     else:
         assert False, type(t)
+
+
+def make_param(name: str, in_: str, type_: TYPE, required: bool, description: Optional[str]) -> JSON:
+    """
+    >>> from azul.doctests import assert_json
+    >>> assert_json(make_param('foo', 'query', int, False, 'razzle that dazzle'))
```

Comment:
Because of the large number of heterogeneous positional args, this invocation is hard to read if it were to occur in an actual spec. To make it easier to read, one would need to pass most of the positionals as keywords anyway which wouldn't be much shorter than the raw JSON literal. OTOH, having this helper enables the IDE's type checker to warn about missing arguments and type mismatches. I think we should leave this helper in, but make it protected and convert all calls to use keyword argument passing even for positional arguments.